qa/workunits/rbd/rbd_mirror_helpers.sh

```diff
@@ -1389,6 +1407,58 @@ get_clone_format()
              }'
 }
 
+list_omap_keys()
+{
+    local cluster=$1
+    local pool=$2
+    local obj_name=$3
+
+    rados --cluster ${cluster} -p ${pool} listomapkeys ${obj_name}
+}
+
+count_omap_keys_with_filter()
+{
+    local cluster=$1
+    local pool=$2
+    local obj_name=$3
+    local filter=$4
+
+    list_omap_keys ${cluster} ${pool} ${obj_name} | grep -c ${filter}
+}
+
+expect_no_omap_keys()
```

Comment:
nit: I would rename it to `wait_for_omap_keys`. And the same for `expect_no_image_in_omap`.